thrust/system/cuda/detail/detail/set_operation.inl

```diff
@@ -302,8 +302,8 @@ inline __device__
       // load the input into __shared__ storage
       __shared__ uninitialized_array<value_type, block_size * work_per_thread> s_input;
   
-      value_type *s_input_end1 = thrust::system::cuda::detail::block::copy_n(ctx, first1, subpartition_size.first,  s_input.begin());
-      value_type *s_input_end2 = thrust::system::cuda::detail::block::copy_n(ctx, first2, subpartition_size.second, s_input_end1);
+      value_type *s_input_end1    =    thrust::system::cuda::detail::block::copy_n(ctx, first1, subpartition_size.first,  s_input.begin());
+      /* value_type *s_input_end2 = */ thrust::system::cuda::detail::block::copy_n(ctx, first2, subpartition_size.second, s_input_end1);
```

Comment:
Let's just eliminate `s_input_end2` and instead of introducing commented-out code. I'm afraid it will just raise questions over why this variable is commented out.